src/main/kotlin/org/jitsi/nlj/rtp/TransportCcEngine.kt

```diff
@@ -135,7 +139,10 @@ class TransportCcEngine(
                     when (packetDetail.state) {
                         PacketDetailState.unreported, PacketDetailState.reportedLost -> {
                             if (packetDetail.state == PacketDetailState.reportedLost) {
-                                numPacketsReportedAfterLost.getAndIncrement()
+                                numPacketsReportedAfterLost.increment()
+                                numPacketsReportedLost.decrement()
+                            } else {
+                                numPacketsReported.increment()
```

Comment:
`numPacketsReported` seems like it should always get incremented?  Or would this packet have been counted in another path/at another time already?

Reply:
Yes, `numPacketsReported` is incremented for this packet when its state goes from `unreported` to `reportedLost`.